lib/src/component_declaration/component_type_checking.dart

```diff
@@ -243,6 +244,11 @@ bool isComponentOfType(ReactElement instance, dynamic typeAlias, {
     return false;
   }
 
+  // When a component is wrapped in a react.memo, we can gain access to the
+  // original Dart component via the 'WrappedComponent` property.
+  if (instance.type != null && getProperty(instance.type, 'WrappedComponent') != null) {
+    instanceType = getProperty(instance.type, 'WrappedComponent');
+  }
```

Comment:
Hm, I just took this out; was this a bad merge conflict, or was this added back in intentionally?

Reply:
Nope, I definitely didn't mean to add that back in. Sorry about that!